packages/teleport/src/features.ts

```diff
@@ -376,10 +377,37 @@ export class FeatureTrust {
   }
 }
 
+export class FeatureDatabases {
+  getTopNavTitle() {
+    return 'Databases';
+  }
+
+  route = {
+    title: 'Databases',
+    path: cfg.routes.databases,
+    exact: true,
+    component: Databases,
+  };
+
+  register(ctx: Ctx) {
```

Comment:
db feature is similar to apps where it can be enabled/disabled depending on the users RBAC rules. Their role can say `disable database feature`, then we shouldn't allow user to access this feature on UI. Examine how Apps is being registered